### PR TITLE
Add 2023 Utah Tax Rules

### LINKED ISSUE #4198: Add 2023 Utah Tax Rules
2023 Income Tax Form Instructions: https://tax.utah.gov/forms/current/tc-40inst.pdf

### PR DESCRIPTION
**Pull request by JudyZ13**
Fixes #4198

- Added the references of 2023 Tax Form
- Updated the 2023 values of ss_benifits phase-out threshold
- Updated the 2023 values of taxpayer credit personal exemption
- Updated the 2023 values of taxpayer credit phase-out threshold

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ut/tax/income/credits/taxpayer/personal_exemption.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ description: This exemption is added to federal standard or itemized deductions
 values:
   2021-01-01: 1_750
   2022-01-01: 1_802
+  2023-01-01: 1_941
 metadata:
   unit: currency-USD
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ut/tax/income/credits/taxpayer/personal_exemption.yaml:7]:
```suggestion
  unit: currency-USD
  uprating: gov.irs.uprating
  period: year
```

File: policyengine_us/parameters/gov/states/ut/tax/income/credits/taxpayer/phase_out/threshold.yaml [hunk 2]
```diff
@@ -2,18 +2,23 @@ description: The Utah taxpayer credit is phased out for taxpayers with incomes a
...
 SEPARATE:
   2021-01-01: 15_095
   2022-01-01: 15_548
+  2023-01-01: 16_742
 SURVIVING_SPOUSE:
   2021-01-01: 30_190
   2022-01-01: 31_096
+  2023-01-01: 33_484
 metadata:
   unit: currency-USD
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ut/tax/income/credits/taxpayer/phase_out/threshold.yaml:23]:
```suggestion
  unit: currency-USD
  uprating: gov.irs.uprating

```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: changelog_entry.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - Add 2023 Utah Tax Rules.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - 2023 Utah Tax Rules.
```

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: could you add the reference to policyengine_us/parameters/gov/states/ut/tax/income/rate.yaml

Comment (PavelMakarchuk) [reply changelog_entry.yaml:1]: Could you also check this piece:
image

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: changelog_entry.yaml
Comment (PavelMakarchuk) [reply changelog_entry.yaml:1]: What would be the difference to the current ut_personal_exemption?

Comment (JudyZ13) [reply changelog_entry.yaml:1]: image
I think there is not any difference? In the tax form it just sums up all the qualifying dependents and I think the current ut_personal_exemption does the same?

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/parameters/gov/states/ut/tax/income/credits/at_home_parent/amount.yaml [hunk 4]
```diff
@@ -10,3 +10,5 @@ metadata:
       href: https://le.utah.gov/xcode/Title59/Chapter10/59-10-S1005.html
     - title: 2021 Utah Indivdual Income Tax Form
       href: https://www.taxformfinder.org/forms/2021/2021-utah-tc-40-full-packet.pdf#page=23
+    - title: 2023 Utah Indivdual Income Tax Form
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ut/tax/income/credits/at_home_parent/amount.yaml:13]: and elsewhere
```suggestion
    - title: 2023 Utah Individual Income Tax Form
```

File: policyengine_us/parameters/gov/states/ut/tax/income/credits/taxpayer/phase_out/threshold.yaml [hunk 5]
```diff
@@ -2,24 +2,32 @@ description: The Utah taxpayer credit is phased out for taxpayers with incomes a
...
+  uprating: gov.irs.uprating
   period: year
   label: Utah taxpayer credit phase-out threshold
   reference:
     - title: Utah Form TC-40 (2022) line 17
       href: https://tax.utah.gov/forms/current/tc-40.pdf#page=1
     - title: Utah Form TC-40 (2021) line 17
       href: https://www.taxformfinder.org/forms/2021/2021-utah-tc-40-full-packet.pdf#page=10
+    - title: 2023 TC-40  Utah Individual Income Tax Form (Line 17)
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ut/tax/income/credits/taxpayer/phase_out/threshold.yaml:32]: add legal code reference

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**